src/main/java/net/atos/client/klant/model/Actor.java

```diff
@@ -1,13 +1,17 @@
 /*
- * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-FileCopyrightText: 2025 Lifely
```

Comment:
No need to update the year. This year just means the first year when we started working on it.

Reply:
Fixed to 2024